src-docs/spec/selfAddr_and_selfName.md

```diff
@@ -1,27 +1,73 @@
 # selfAddr & selfName
 
+## selfName
+
+```js
+window.webxdc.selfName
+```
+
+`selfName` is the nick or display name for the user 
+which can be displayed in the user interface of the webxdc application. 
+
+
 ## selfAddr
 
 ```js
 window.webxdc.selfAddr
 ```
 
-A string with an unique ID identifying the user in the current webxdc.
-Every user of an webxdc must get a different ID
-and that ID must be the same if the webxdc is started again later for the same user.
-The same user in different webxdc, however, may have different IDs.
+`selfAddr` is a unique string within a webxdc application that
 
-Especially useful if you want to differentiate between different peers -
-just send the ID along with the payload,
-and, if needed, compare the payload addresses against `selfAddr` later on.
+- should not be shown in the user interface of the webxdc application,
 
-## selfName
+- can be used in other webxdc APIs,
+
+- is identical across multiple invocations of the same webxdc application,
+
+- is identical on multiple devices of the user using the same webxdc application, 
+
+- should not be linkable to addresses of the same user in other webxdc applications. 
+
+
+## Example using selfAddr and selfName
+
+Here is a simple chat app that sends out a reply using the display names
+but uses the addresses for notifications. 
 
 ```js
-window.webxdc.selfName
-```
 
-Name of the current account, as defined in settings.
-If empty, this defaults to the peer's address.
+// Receive a message from anyone in the chat 
+let users = new Set();
+
+setUpdateListener((update) => {
+    let prompt = "${update.payload.senderName} (${update.payload.senderAddr}):";
```

Comment:
everywhere in the code where "" is used with variables inside, must be changed to:

```
`${var} example`
```

`"` is for literal strings and doesn't support format-string syntax

Reply:
did this, thanks